user-guide/lobby-rules.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: default
+title: Lobby Rules
+permalink: /user-guide/lobby-rules
+---
+
+
+### General Rules
+
+#### Use the golden rule, "Treat others as you would want to be treated.”
+
+We have 5 rules written as follows, all players, including admins, must follow these simple 
+rules when playing on  the lobby, in bots, and when using the forums;
+
+* Follow the Golden Rule and treat others as you would like to be treated!
+
+* No racism, sexism, and homophobic, or otherwise, vulgar language will be tolerated. 
+The admins do not care if it does not offend you and/or your friends. No bullying and/or 
+discrimination will be tolerated, that includes; Race, Religion, Sex, Age, Cultural Background, 
+Sexual Orientation. Use of foul language, discrimination and derogatory comments are not acceptable, 
+and are subject to admin disciplinary action. It is up to the admin to decide what is or is not 
+acceptable in regards to defamation, discrimination and spamming.
+
+* Due to the recent spat of BLATANT! racism in the lobby the TripleA Council has decided to increase 
+the punishment for these offences to an instant 2 week Ban on first offense!
+
+* No verbal abuse. Doing so violates rule #1 above.
```

Comment:
\# might need escaping, just markdown things

Reply:
Good consideration, happily only if the '#' is at start of line, seems to render okay:

![Screenshot from 2019-06-08 13-10-49](https://user-images.githubusercontent.com/12397753/59151843-e218ab00-89ee-11e9-9703-cb9e183360fc.png)